erna/__init__.py

```diff
@@ -18,9 +18,25 @@ def build_path(row, path_to_data, extension):
     year = night[0:4]
     month = night[4:6]
     day = night[6:8]
-    return os.path.join(path_to_data, year, month, day, row.filename + extension)
+    res = os.path.join(path_to_data, year, month, day, row.filename + extension)
+    if not os.path.exists(res):
+        return np.nan
+    return res
```

Comment:
I would suggest to rather keep this function as it was and doing the check if a file exist in a separate function.

Reply:
If you do so, it would be good to log which or how many files could not be found

Reply:
This is now implemented in pyfact. `fact.path`

Reply:
awesome